### ISSUE TITLE
/api/v2/seamail/:id sends back a thread rather than a message

### ISSUE THREAD

**Issue by Hixie** [seamonkeysocial/twitarr#287]
The docs say that when you post to a thread you should get back a JSON payload with a message, but right now the server sends back the whole thread:

```
I/flutter (12442): >>> POST http://69.62.137.54:7001/api/v2/seamail/3?key=test2%3A1579935322071%3A3d7a8acff800af2e588b5d781eee637bb0f931e5&app=plain with payload of length 14:
I/flutter (12442):  0x00000: {"text":"xxx"}
I/flutter (12442): <<< 200 from /api/v2/seamail/3?key=test2%3A1579935322071%3A3d7a8acff800af2e588b5d781eee637bb0f931e5&app=plain:
I/flutter (12442):  0x00000: {"status":"ok","seamail_message":{"id":"3","users":[{"username":"kvort","display_name":"kvort","last_photo_updated":157903570059
I/flutter (12442):  0x00080: 8},{"username":"test2","display_name":"test2","last_photo_updated":1579067622027}],"subject":"kvort and test2","messages":[{"id"
I/flutter (12442):  0x00100: :"6","author":{"username":"test2","display_name":"test2","last_photo_updated":1579067622027},"text":"test","timestamp":157907061
I/flutter (12442):  0x00180: 6398},{"id":"7","author":{"username":"test2","display_name":"test2","last_photo_updated":1579067622027},"text":"hello","timestam
I/flutter (12442):  0x00200: p":1579070620813},{"id":"8","author":{"username":"test2","display_name":"test2","last_photo_updated":1579067622027},"text":"hell
I/flutter (12442):  0x00280: o","timestamp":1579070669484},{"id":"9","author":{"username":"test2","display_name":"test2","last_photo_updated":1579067622027},
I/flutter (12442):  0x00300: "text":"test","timestamp":1579070815793},{"id":"10","author":{"username":"test2","display_name":"test2","last_photo_updated":157
I/flutter (12442):  0x00380: 9067622027},"text":"test","timestamp":1579070984736},{"id":"11","author":{"username":"test2","display_name":"test2","last_photo_
I/flutter (12442):  0x00400: updated":1579067622027},"text":"test","timestamp":1579071105288},{"id":"12","author":{"username":"test2","display_name":"test2",
I/flutter (12442):  0x00480: "last_photo_updated":1579067622027},"text":"hello","timestamp":1579071194428},{"id":"13","author":{"username":"test2","display_n
I/flutter (12442):  ...(continues for 318 more bytes)
```

Compare to posting on the old server:
```
I/flutter (30106): >>> POST https://twitarr.wookieefive.net/api/v2/seamail/5e1a1fcb25692d000126a2a4?key=hixie%3A1579935340253%3A51434bbbbca573713d342d9d323d2cca8724774c&app=plain with payload of length 14:
I/flutter (30106):  0x00000: {"text":"xxx"}
I/flutter (30106): <<< 200 from /api/v2/seamail/5e1a1fcb25692d000126a2a4?key=hixie%3A1579935340253%3A51434bbbbca573713d342d9d323d2cca8724774c&app=plain:
I/flutter (30106):  0x00000: {"status":"ok","seamail_message":{"id":"5e1eb7fd25692d000126b45a","author":{"username":"hixie","display_name":"Hixie","last_phot
I/flutter (30106):  0x00080: o_updated":1576560200024},"text":"xxx","timestamp":1579071485487,"read_users":[{"username":"hixie","display_name":"Hixie","last_
I/flutter (30106):  0x00100: photo_updated":1576560200024}]}}
```

**Comment by Hixie** [comment]
cc @hendricksond

**Comment by hendricksond** [comment]
Fixed by #294
